tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -256,7 +260,7 @@ def test_positive_generate_report_hoststatus_filter(session):
 @tier2
 @stubbed()
 def test_positive_generate_report_hoststatus_sanitized(session):
-    """ Edit a report template
+    """ Generate a report for some host with comma in some field
```

Comment:
what is this testing? what field the comma put to?

Reply:
This tests if the comma in some field (in steps, it is specified it's gonna be a OS field but it doesn't matter much) doesn't break the resulting CSV.

This is not related to the new feature, there was bad description and I just fixed it as part of the commit.

Reply:
Ah, got it, some value that us printed to the report will contain comma. Ok makes sense, thank you